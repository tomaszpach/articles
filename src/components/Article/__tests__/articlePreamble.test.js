import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import ArticlePreamble from "../articlePreamble";

describe("ArticlePreamble", () => {
  test("should correctly render ArticlePreamble component", () => {
    const wrapper = shallow(<ArticlePreamble preamble="lorem ipsum dolor sit amet" />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
