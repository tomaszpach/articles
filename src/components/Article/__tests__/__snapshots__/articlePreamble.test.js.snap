// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticlePreamble should correctly render ArticlePreamble component 1`] = `
<p>
  lorem ipsum dolor sit amet
</p>
`;
